index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Wasyihun-portfolio</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div>
+        <a class="My_logo">MyLogo</a>
+      </div>
+      <nav class="menu_icon">
+        <div class="menu_icon"></div>
+        <div class="menu_icon"></div>
+        <div class="menu_icon"></div>
+      </nav>
+    </header>
+    <section class="Body_content">
+      <div class="Body_content_title">
+        <h1>
+          I’m Wasyihun<br />
+          Glad to see<br />
+          you!
+        </h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h2>LET’S CONNECT</h2>
+        <ul>
+          <li><img src="image/Twittwer_vector.png" alt="" /></li>
+          <li><img src="image/Linkedin_icon.png" alt="" /></li>
+          <li><img src="image/M_vector.png" alt="" /></li>
+          <li><img src="image/Github_vector.png" alt="" /></li>
+          <li><img src="image/Pieace_vector.png" alt="" /></li>
+        </ul>
+      </div>
+    </section>
+    <section class="Works">
+      <main class="Works_main">
+        <img class="big_pictures" src="image/Picture_1.png" alt="" />
+        <h2>Tonic</h2>
+        <ul class="Works_Tony">
+          <li>CANOPY</li>
+          <li><img src="image/Card_1.png" alt="" /></li>
+          <li>Back End Dev</li>
+          <li><img src="image/Card_1.png" alt="" /></li>
+          <li>2015</li>
+        </ul>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
```

Comment:
The size dimensions of your button are smaller than the ones given in Figma. Kindly see the comparison.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219757765-a74d5e74-0f82-4cb2-8540-d2ddef83ee3b.png)  | ![image](https://user-images.githubusercontent.com/61361037/219757522-95b6b759-276f-4d3f-aea7-f47f6ea117d8.png)  |